src/Modules/EnvGenerator/Generator.php

```diff
@@ -95,7 +95,7 @@ protected function prepare_variable( string $name, ?string $value ): ?string {
 		$required    = ! empty( $variable['required'] );
 
 		// Check if a required variable has a value.
```

Comment:
This is waay beyond the scope right now, but it also helps explain why we're currently excluding `'0'`.

```suggestion
		// Check if a required variable has a value.
		// @todo: handle NODE_TLS_REJECT_UNAUTHORIZED by checking the NEXT_PUBLIC_URL
```